packages/components/src/Select/Select.stories.tsx

```diff
@@ -87,13 +88,13 @@ export const Basic: StoryObj<typeof Select> = {
     return (
       <>
         <Select {...args} onChange={setSelected} disabledKeys={['Firefly']}>
-          <Select.Option key="Harry Potter">Harry Potter</Select.Option>
-          <Select.Option key="Lord of the Rings">
+          <Select.Option id="Harry Potter">Harry Potter</Select.Option>
+          <Select.Option id="Lord of the Rings">
             Lord of the Rings
           </Select.Option>
-          <Select.Option key="Star Wars">Star Wars</Select.Option>
-          <Select.Option key="Star Trek">Star Trek</Select.Option>
-          <Select.Option key="Firefly">Firefly</Select.Option>
+          <Select.Option id="Star Wars">Star Wars</Select.Option>
```

Comment:
Same issue like in the menu @sebald mentioned: "the items now have "id" but the props are still called "key" should we rename them?"

Reply:
Ok I think it is already decieded, so never mind. https://github.com/marigold-ui/marigold/pull/3546/files